thunder/tests/test_dynamo.py

```diff
@@ -6,6 +6,7 @@
 
 import torch
 import pytest
+from unittest.mock import patch
```

Comment:
noob question: where is this `patch` utilized?

Reply:
Good catch, I forgot to remove this. I had initially used it to disable `LazyGraphModule` usage but later found that there was a ctx manager. Thanks!